Members update 2022

kylemtam opened the pull request:
- Delete Aiden, Ryan, Waleed, Ronald, Sophie, Kaelan, Tommy, Khanh, Nadine, Derek, Dhruv, Alex, Ximing, Soha, Daniel
- Add Joel, Kimi, Denis, Stefan, Andre, Adam, Charlie, Jo
- Revise a number of existing members' titles and photos
- [https://docs.google.com/spreadsheets/d/1OhANTTXS00mJcnI6krWE5CQDB5j4x338YweHgBvXB1A/edit?usp=sharing](https://docs.google.com/spreadsheets/d/1OhANTTXS00mJcnI6krWE5CQDB5j4x338YweHgBvXB1A/edit?usp=sharing)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/waterloo-rocketry/website/138)
<!-- Reviewable:end -->

kylemtam committed:
Update current members

Delete old members

kylemtam committed:
Add new members

- Update existing members' photos

kylemtam committed:
Correct team leads title

kylemtam committed:
Update rio.jpg

QuantumManiac reviewed (changes requested):
Overall, looks good except for that one duplicate image. I'm seeing a lot of pictures getting quite large - we definitely want to be compressing/optimizing these images. Will probably put up a separate PR to compress all the pictures on the site after this is merged if that's fine.

> Reviewed 34 of 34 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/website/138#-:-Mz7WVGV06J0ezIJCY-f:bunu6v7)* status: all files reviewed, 1 unresolved discussion (waiting on @chrissank, @kylemtam, and @wendi-yu)

___
*[images/members/lana.JPG, line 0 at r1](https://reviewable.io/reviews/waterloo-rocketry/website/138#-Mz7WM57B8Sx58mkVEAW:-Mz7WM58Dijlq3NNkUSX:b-rsmk39) ([raw file](https://github.com/waterloo-rocketry/website/blob/cdc32ca63c85b67b6a96bd5a8a5886b2c213fc62/images/members/lana.JPG#L0)):*
Why is there a `lana.jpg` and a `lana.JPG`?


<!-- Sent from Reviewable.io -->

kylemtam committed:
Delete lana.JPG

kylemtam committed:
Delete lana.jpg

kylemtam committed:
Create lana.jpg

kylemtam committed:
Compress photos

kylemtam committed:
Fix Artem and Kimi

kylemtam committed:
Crop Stefan

kylemtam commented:
I think I've fixed the duplicate image thing. For some reason it wasn't showing up on my Windows machine, so I just deleted both on github and then readded it on my computer. I've also compressed some of the newer photos.

QuantumManiac commented:
> I think I've fixed the duplicate image thing. For some reason it wasn't showing up on my Windows machine, so I just deleted both on github and then readded it on my computer. I've also compressed some of the newer photos.

iirc Windows does not support case-sensitive file extensions, so you probably ended up seeing only one of the two files.
